client/src/actions/interviewerActions.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
```

Comment:
If you're just fetching the list of interviewers, we already have that in userActions

Reply:
directoryActions*